nautobot/utilities/utils.py

```diff
@@ -147,10 +147,16 @@ def serialize_object_v2(obj):
     """
     Return a JSON serialized representation of an object using obj's serializer.
     """
+    from nautobot.core.api.exceptions import SerializerNotFound 
```

Comment:
Does this have to be a local import to avoid a loop, or can we add it to the top of the file instead?

Reply:
Yes, It has to be a local import